tests/wpt/webgl/meta/conformance2/misc/object-deletion-behaviour-2.html.ini

```diff
@@ -1,5 +1,17 @@
 [object-deletion-behaviour-2.html]
   expected: ERROR
+  [WebGL test #12: gl.isBuffer(bufferBaseTransformFeedback) should be false. Was true.]
```

Comment:
Judging by https://github.com/servo/servo/blob/52fd6f8aa848699f8f1d40264dd66a765b30dbfa/components/script/dom/webglbuffer.rs#L127, this new failure appears to be caused by the attachment count having a different value after these changes.